.evergreen/resync-specs.sh

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+# exit when any command fails
+set -e
+PYMONGO=~/Work/mongo-python-driver
+SPECS=~/Work/specifications
+
+help (){
+echo "Usage: resync_specs.sh [-bcsp] spec"
+echo "Required arguments:"
+echo " spec     determines which folder the spec tests will be copied from."
+echo "Optional flags:"
+echo " -b  is used to add a string to the blocklist for that next run. Can be used"
+echo "     any number of times on a single command to block multiple patterns."
+echo "     You can use any regex pattern (it is passed to 'grep -Ev')."
+echo " -c  is used to set a branch or commit that will be checked out in the"
+echo "     specifications repo before copying."
+echo " -s  is used to set a unique path to the specs repo for that specific"
+echo "     run."
+echo " -p  does the same thing but for the pymongo repo."
+}
+
+# Parse flag args
+BRANCH=""
+BLOCKLIST='.*\.yml'
+while getopts 'b:c:s:p:' flag; do
+  case "${flag}" in
+    b) BLOCKLIST+="|"$OPTARG""
+      ;;
+    c) BRANCH="${OPTARG}"
+      ;;
+    s) SPECS="${OPTARG}"
+      ;;
+    p) PYMONGO="${OPTARG}"
+      ;;
+    *) help; exit 0
+      ;;
+  esac
+done
+shift $((OPTIND-1))
+
+if [ -z $BRANCH ]
+then
+git -C $SPECS checkout $BRANCH
+fi
+
+# Ensure the JSON files are up to date.
+cd $SPECS/source
+make
+cd -
+# cpjson2 unified-test-format/tests/invalid unified-test-format/invalid
+# * param1: Path to spec tests dir in specifications repo
+# * param2: Path to where the corresponding tests live in Python.
+cpjson2 () {
```

Comment:
Rename to cpjson since there is no cpjson1 anymore.

Reply:
Done.